src/ucp/rndv/proto_rndv.c

```diff
@@ -427,13 +434,14 @@ static void ucp_proto_rndv_recv_completion(void *request, ucs_status_t status,
 }
 
 void ucp_proto_rndv_receive(ucp_worker_h worker, ucp_request_t *recv_req,
-                            const ucp_rndv_rts_hdr_t *rts, size_t hdr_len)
+                            const ucp_rndv_rts_hdr_t *rts,
+                            const void *rkey_buffer, size_t rkey_length)
 {
-    const void *rkey_buffer;
     ucs_status_t status;
     ucp_request_t *req;
     uint8_t sg_count;
     size_t length;
+    int has_rkey;
```

Comment:
can use `rkey_length`, why need another var?

Reply:
removed; added assert to make sure logic is not changed